pegasus/sites.v3/code.org/public/teacher-dashboard/sections.haml

```diff
@@ -155,3 +156,5 @@ content-type: text/ng-template
   %br
     %p= I18n.t('dashboard_privacy_document_link')
   %br/
+
+%script{src: 'https://apis.google.com/js/api.js'}
```

Comment:
Good call, I will fix.  One downside is that I'll need to import this in `teacher-dashboard/index.js` so it'll get pulled into that package (and loaded even on pages that don't need it).